src/components/swap/UnsupportedCurrencyFooter.tsx

```diff
@@ -100,7 +102,7 @@ export default function UnsupportedCurrencyFooter({
           </AutoColumn>
         </Card>
       </Modal>
-      <ButtonEmpty padding={'0'} onClick={() => setShowDetails(true)}>
+      <ButtonEmpty padding={'0'} style={{ textDecoration: 'none' }} onClick={() => setShowDetails(true)}>
```

Comment:
Why is this bit necessary?

Reply:
I generally pref avoiding inline styles in favor of [modifying the base styled-component](https://styled-components.com/docs/basics#extending-styles).

Reply:
this was a request from @willhennessy to remove the underline. 

@JFrankfurt agree with you there but the base component is used a few other places (https://github.com/Uniswap/uniswap-interface/blob/main/src/components/PositionCard/V2.tsx#L99) and also has some inline styles there, so I'm hesitant to modify it without knowing the cascading effects

one thing that helps me feel a little more comfortable modifying base component styles is jest snapshot tests - have y'all considered those at all?

Reply:
I think Jordan meant styling the base component in this file

```
const StyledButtonEmpty = styled(ButtonEmpty)`
  text-decoration: none;
`
```

Reply:
i recently added support for jest snapshots but our coverage is almost 0 right now.

https://github.com/Uniswap/uniswap-interface/tree/main/src/components/TextInput